[ADD]-project_issue_helpdesk_invoicing

carlosnavarro24 opened the pull request:
This module permit add additional cost in issues for provisions.
This provisions are editable in the partner form.
The issues invoicing add a new lines for this additional costs.

carlosnavarro24 committed:
[ADD]-project_issue_helpdesk_invoicing

This module permit add additional cost in issues for provisions.
This provisions are editable in the partner form.
The issues invoicing add a new lines for this additional costs.

ghost commented:
Check project_issue_helpdesk_invoicing/wizard/project_issue_helpdesk_invoice_wizard.py
Please separate the code in methods, currently is not maintainable
Review conditions in order to minimize them

carlosnavarro24 committed:
[FIX]-Separate code in methods for invoicing issues

Add separate methods for invoicing:
Timesheets
BackOrders
Supplier Invoices
Additional Costs
Expenses

Test of conditions of invoicing in invoice wizard

carlosnavarro24 commented:
Add commit with changes in project_issue_helpdesk_invoicing/wizard/project_issue_helpdesk_invoice_wizard.py. Te code of issues invoicing was separated in methods, and update conditions

carlosnavarro24 committed:
[ADD]-project_issue_helpdesk_invoicing-Add comments to methods of issues invoicing wizard

carlosnavarro24 commented:
Add comments to methods for invoices issues wizard
